lib/ansible/module_utils/facts/system/distribution.py

```diff
@@ -332,6 +332,13 @@ def parse_distribution_file_Debian(self, name, data, path, collected_facts):
                     rc, out, err = self.module.run_command(cmd)
                     if rc == 0:
                         debian_facts['distribution_release'] = out.strip()
+            debian_version_path = '/etc/debian_version'
+            distdata = get_file_content(debian_version_path)
+            if distdata:
+                for line in distdata.splitlines():
+                    m = re.search(r'(\d+)\.(\d+)', line.strip())
+                    if m:
+                        debian_facts['distribution_minor_version'] = m.groups()[1]
```

Comment:
This could use `get_file_lines()`.
```suggestion
            distdata = get_file_lines(debian_version_path)
            for line in distdata:
                m = re.search(r'(\d+)\.(\d+)', line.strip())
                if m:
                    debian_facts['distribution_minor_version'] = m.groups()[1]
```